requirements/extras/cassandra.txt

```diff
@@ -1 +1 @@
-cassandra-driver<3.24.0
+cassandra-driver<3.21.0
```

Comment:
why reducing this? why not the latest version?

Reply:
It caused this error(Cassandra integration test hang)

if cassandra-driver >= 3.21.0, it will cause a deadlock during integration test

I didn't find the reason

I will continue to find the reason， and then upgrade the version